submarine-workbench/submarine-server/src/main/java/org/apache/submarine/rest/TeamRestApi.java

```diff
@@ -59,37 +59,15 @@ public Response list(@QueryParam("owner") String owner,
                        @QueryParam("pageSize") int pageSize) {
     LOG.info("TeamRestApi.list() owner:{}, pageNo:{}, pageSize:{}", owner, pageNo, pageSize);
 
-    // mock data
     List<Team> teams = new ArrayList<>();
-    for (int i = 0; i < 3; i++) {
-      Team team = new Team();
-      team.setId("team" + i);
-
-      // test different owner
-      if (i == 0) {
-        team.setOwner("admin");
-      } else if (i == 1) {
-        team.setOwner("liuxun");
-      } else {
-        team.setOwner("test");
-      }
-
-      team.setTeamName("test-team" + i);
-
-      for (int j = 0; j < 3; j++) {
-        TeamMemeber memeber = new TeamMemeber();
-        memeber.setId("team" + i + "name" + j + "-id");
-        memeber.setMember("team" + i + "name" + j);
-        if (j == 0) {
-          memeber.setInviter(0);
-        } else {
-          memeber.setInviter(1);
-        }
-        team.addCollaborator(memeber);
-      }
-      teams.add(team);
+    try {
+      // TODO(zhulinhao): Front need to correct 'owner' value, and Whether need the
```

Comment:
We should remove the author name from TODO labels.

Reply:
@jiwq , We set in the TODO style check, who needs to complete TODO.
In this way, we can see which tasks are required to be completed in IDEA's TODO ListView.
https://github.com/apache/hadoop-submarine/blob/master/dev-support/maven-config/checkstyle.xml#L44

Reply:
@liuxunorg Okay, thanks for your help.